src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BooksList from '../container/BooksList';
+import BooksForm from '../container/BooksForm';
+
+function App(store) {
```

Comment:
We don't really need to pass in `store` in here,  you already passed it on `Provider` tag in `index.js` file
Am I missing something?